core/windows_and_tabs/window_snap.py

```diff
@@ -302,14 +328,170 @@ def __init__(self, left, top, right, bottom):
 }
 
 
+def _snap_next(windows: list[Any], target_layout: RelativeScreenPos) -> Optional[int]:
+    print(windows)
```

Comment:
Would recommend you remove your debugging code if you feel confident of its quality.